v-next/hardhat-keystore/src/internal/types-validation.ts

```diff
@@ -0,0 +1,11 @@
+import type { UnencryptedKeystoreFile } from "./types.js";
+import type { ZodSchema } from "zod";
+
+import { z } from "zod";
+
+export const unencryptedKeystoreFileSchema: ZodSchema<UnencryptedKeystoreFile> =
+  z.object({
+    _format: z.literal("hh-unencrypted-keystore"),
+    version: z.number(),
```

Comment:
Should this be:

```
_format: string,
_version: number
```

Or combine into `_format`?

Reply:
It can be deferred for the full encrypted version.

Reply:
+1 on defering

Reply:
_format and _version removed [here](https://github.com/NomicFoundation/hardhat/pull/5654/commits/96c7c1caf690bc711f70bd9f55f1b991249ddab2)

Reply:
Wait, why? I think they made sense The deferred decision was whether they should be one or two fields.

Reply:
oh I though we were planning to introduce them later, I'll revert the commit

Reply:
Btw, should we use `_version` instead of `version`?